### PR TITLE
RootFS: Add no-lldb option to fix arm64 build

### PR DESCRIPTION
**Pull request by qmfrederik**
#16182 and #13200 added lldb to the rootfs for CoreFX.

However, CoreFX does not have a dependency on liblldb (CoreCLR does), and this package is not available for ubuntu.16.04-arm64 and earlier, causing the RootFS generation to fail.

This PR adds an option to disable adding the LLDB package for the CoreFX RootFS.

### CODE REVIEW

**Comment by hqueue** [comment]
I also confirm that lldb is not required for CoreFX.
However, if possible, I prefer alternative way to keep same rootfs and script itself across repo, i.e. adding `no-lldb`.
In addition, make `no-lldb` as default for arm64 will be convenient for developers who are woking on arm64.

File: cross/build-rootfs.sh [hunk 1]
```diff
@@ -101,11 +100,6 @@ for i in "$@" ; do
     esac
 done
 
-if [ "$__BuildArch" == "armel" ]; then
-    __LLDB_Package="lldb-3.5-dev"
-fi
-__UbuntuPackages+=" ${__LLDB_Package:-}"
```
Comment (hqueue) [inline cross/build-rootfs.sh]: How about  `__LLDB_Package=""` here for arm64 similar to armel ?

**Comment by qmfrederik** [comment]
@hqueue OK, I updated the PR to make the script mirror the CoreCLR script. There's now a no-lldb option.

I prefer not to unset lldb for ARM64, because Ubuntu Zesty does include lldb for ARM64.

**Comment by gkhanna79** [comment]
@qmfrederik LGTM. Is this good to merge? BTW, we should get this "no-lldb" support to Core-Setup as well for consistency.

**Comment by qmfrederik** [comment]
@gkhanna79 I think this is good to merge - basically brings the build-rootfs script in line with what we have in  coreclr.

**Comment by qmfrederik** [comment]
@dotnet-bot test Innerloop CentOS7.1 Debug Build and Test please

**Comment by qmfrederik** [comment]
@dotnet-bot test Innerloop Windows_NT Release Build and Test please

**Pull request merged**